Source/Fuse.Controls.ScrollView/ScrollView.uno

```diff
@@ -377,7 +377,12 @@ namespace Fuse.Controls
 		*/
 		public void GotoRelative( float2 position )
 		{
-			Goto( (MaxScroll - MinScroll) * position + MinScroll );
+			Goto( RelativeToAbsolutePosition(position) );
+		}
+		
+		public float2 RelativeToAbsolutePosition( float2 pos )
```

Comment:
To make it a bit clear, I think "absolute position" should refer to a position that isn't relative to *anything*, not just "not relative to scroll position"...